packages/bpk-component-card/src/BpkCardWrapper.module.scss

```diff
@@ -0,0 +1,61 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index.scss';
+
+.bpk-card-wrapper {
+  @include bpk-card;
+  @include bpk-border-radius-md;
+  @include bpk-border-lg(var(--background-color), '');
+
+  &--header {
+    border-radius: $bpk-border-radius-sm() $bpk-border-radius-sm 0 0;
```

Comment:
I think there's an extra set of parenthesis for the top left border radius?

Reply:
nice catch, it's  extra set of parenthesis